corehq/apps/app_manager/detail_screen.py

```diff
@@ -325,12 +325,17 @@ class EnumImage(Enum):
 
 
 @register_format_type('late-flag')
-class LateFlag(HideShortHeaderColumn):
+class LateFlag(FormattedDetailColumn):
 
     template_width = "10%"
 
     XPATH_FUNCTION = u"if({xpath} = '', '*', if(today() - date({xpath}) > {column.late_flag}, '*', ''))"
 
+    @property
+    def header_width(self):
+        if self.detail.display == 'short':
+            return 30  # TODO: Test and adjust
```

Comment:
In that case it would be nice to explicitly `return None` so it doesn't look like someone forgot to finish the function.